### PR TITLE
Simplify descriptions by mentioning partial updates

### PR DESCRIPTION
**Pull request by Boydbueno**
## Checklist

- [x] Writing style
- [x] API design standards

### CODE REVIEW

File: openapi/paths/credit-memos@{id}.yaml [hunk 1]
```diff
@@ -74,11 +74,11 @@ patch:
     - Core
   tags:
     - Credit memos
-  summary: Update credit memo, allocation, or both
+  summary: Partially update a credit memo
```
Comment (nanobash) [inline openapi/paths/credit-memos@{id}.yaml:77]: Maybe with dot?

```suggestion
  summary: Partially update a credit memo.
```

Comment (Boydbueno) [reply openapi/paths/credit-memos@{id}.yaml:77]: Summaries should not be ended with a period.

File: openapi/paths/credit-memos@{id}.yaml [hunk 2]
```diff
@@ -74,11 +74,11 @@ patch:
...
   operationId: PatchCreditMemo
   x-sdk-operation-name: patch
   description: |-
-    Updates a credit memo, credit memo allocation, or both credit memo and allocation with a specified ID.
+    Partially updates a credit memo with specified identifier string.
```
Comment (noon-dawg) [inline openapi/paths/credit-memos@{id}.yaml:81]: nitpick: I think it reads better with an "a" here
```suggestion
    Partially updates a credit memo with a specified identifier string.
```

Comment (Boydbueno) [reply openapi/paths/credit-memos@{id}.yaml:81]: I couldn't decide between the two! Thanks for the suggestion. :)

**Review by nanobash — APPROVED** [review]

**Comment by noon-dawg** [comment]
LGTM

**Review by vykhrystiuk — APPROVED** [review]

**Pull request merged**